### PR TITLE
Day11_to_Day15, REFS : #91 #92 #95 #96 #98

### PR DESCRIPTION
**Pull request by Kartikraval**

### CODE REVIEW

**Review by MilitPanchasara — COMMENTED** [review]
Day 9 & 10 => Add navigations to all pages.

**Review by MilitPanchasara — COMMENTED** [review]
Day11 => navigation not working, painting modal not working.

**Review by MilitPanchasara — COMMENTED** [review]
Day14 => Please check that all navigations are working properly, also do not use new tabs for those which are not the Layouts.
Day15 => In assignment ,Clear button not working
Day15 => In practice, Only do sum of those numbers which are >40

**Comment by Kartikraval** [comment]
> Day11 => navigation not working, painting modal not working.

Changes done

**Comment by Kartikraval** [comment]
> Day14 => Please check that all navigations are working properly, also do not use new tabs for those which are not the Layouts.
> Day15 => In assignment ,Clear button not working
> Day15 => In practice, Only do sum of those numbers which are >40

changes done

**Comment by Kartikraval** [comment]
> Day 9 & 10 => Add navigations to all pages.

changes done

**Comment by Kartikraval** [comment]
All changes done

Sent from Yahoo Mail on Android 
 
  On Thu, 25 Feb 2021 at 12:00 pm, Milit Panchasara<notifications@github.com> wrote:   


@MilitPanchasara commented on this pull request.

Day14 => Please check that all navigations are working properly, also do not use new tabs for those which are not the Layouts.

File: Modules/SQL/Day2/kartik.raval/Practice_Assignment/Practice9.sql [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+('203', 'Susan', 'Mavris', 'SMAVRIS', '515.123.7777', '1987-09-28', 'HR_REP', '6500.00', '0.00', '101', '40'),
+('204', 'Hermann', 'Baer', 'HBAER', '515.123.8888', '1987-09-29', 'PR_REP', '10000.00', '0.00', '101', '70'),
+('205', 'Shelley', 'Higgins', 'SHIGGINS', '515.123.8080', '1987-09-30', 'AC_MGR', '12000.00', '0.00', '101', '110'),
+('206', 'William', 'Gietz', 'WGIETZ', '515.123.8181', '1987-10-01', 'AC_ACCOUNT', '8300.00', '0.00', '205', '110');
+
+SELECT * FROM Employees8;	
+
+UPDATE Employees8 SET Salary = (Salary + 2000) WHERE JobId = 'PU_CLERK';
+
+UPDATE Employees8 SET CommissionPct = (CommissionPct + .10) where Salary <= 4800;
```
Comment (radixdt12020) [inline Modules/SQL/Day2/kartik.raval/Practice_Assignment/Practice9.sql:129]: use case statement

**Pull request merged**